soccer/src/soccer/planning/params.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <rj_param_utils/param.hpp>
+
+namespace trajectory_utils {
+    constexpr auto kMaxIterations = "trajectory_utils";
+    constexpr auto kExpectedDt = "trajectory_utils";
+
+    DECLARE_INT64(kMaxIterations, max_iterations)
```

Comment:
not sure if I understand this correctly, but if kMaxIterations represents the same value as max_iterations why not just use max_iterations and cut out the middle-man variable

Reply:
i think kMaxIterations is later modified to be a different value? but maybe im just understanding it differently as well.

Reply:
This should be `DECLARE_INT(kPlanningParams, max_iterations)`.

The first argument to this macro is a string representing the "module name", which is just how we group together a bunch of parameters. The second one is the variable name of the parameter, which can be given an initial value but is also able to be changed through ROS.